drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/EdgeDriverBinaryHandler.java

```diff
@@ -108,6 +109,11 @@ public ExternalBinary getReleaseForVersion(String version) throws Exception {
             return new ExternalBinary(driverVersion, webDriverUrl);
         }
 
+        @Override
+        public String getFileNameRegexToDownload(String version) {
+            return null;
```

Comment:
Actually, looking at the code again - it is OK - here https://github.com/arquillian/arquillian-extension-drone/pull/90/files#diff-71c8ade0188a564bdff67fcb52737394R181 I check if the regex is null or not - if it is null, then I don't filter the directory for any files... The reason is that in the case of "Edge" I don't know the name of the file that will be downloaded.